cmd/wait-for/main.go

```diff
@@ -18,11 +18,13 @@ func main() {
 	httpTimeoutParam := "1s"
 	configFile := ""
 	var quiet bool
+	regexStatus := ""
 
 	flag.StringVar(&timeoutParam, "timeout", timeoutParam, "time to wait for services to become available")
 	flag.StringVar(&httpTimeoutParam, "http_timeout", httpTimeoutParam, "timeout for requests made by a http client")
 	flag.StringVar(&configFile, "config", "", "configuration file to use")
 	flag.BoolVar(&quiet, "quiet", false, "reduce output to the minimum")
+	flag.StringVar(&regexStatus, "regex", "", "Use regex to match the expected result in HTTP status codes")
```

Comment:
```suggestion
	flag.StringVar(&statusPattern, "status", "200", "A golang regex that represents the desired HTTP status response code")
```

You'll probably have to rename a few instances of variable used here.

Reply:
I will rename it statusPatternParam , Do you approve?

Reply:
Yes, you're right. That follows the convention.